apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.theme.scss

```diff
@@ -1,15 +1,6 @@
 cde-create-visualization-page {
```

Comment:
Move content into `create-visualization-page.component.scss` and delete this file